flycheck.el

```diff
@@ -6182,6 +6183,19 @@ Requires DMD 2.066 or newer.  See URL `http://dlang.org/'."
             (or "Warning" "Deprecation") ": " (message) line-end))
   :modes d-mode)
 
+(flycheck-define-checker elixir-dogma
+  "An Elixir syntax checker using the Dogma analysis tool.
+
+See URL `https://github.com/lpil/dogma/'."
+  :command ("mix" "dogma" "--format=flycheck" source-inplace)
```

Comment:
By the way how does `mix` know that `dogma` is available?  Can we somehow test that to make sure that we don't enable this syntax checker when `mix` is there but `dogma` is not?